### PR TITLE
UI: add an item 'administration' on the top menu

### PR DESCRIPTION
**Pull request by laurentj**
Refs #1044

### CODE REVIEW

**Comment by laurentj** [comment]
In the map page:
image

In an administration page:
image

File: lizmap/modules/lizmap/templates/user_menu.tpl [hunk 1]
```diff
@@ -1,5 +1,9 @@
 {hook 'LizTopMenuHtmlItems'}
+
 {ifuserconnected}
+    <li class="dashboard-item"><a href="{jurl 'master_admin~default:index'}">
+            <span class="icon dashboard-icon"></span> <span class="text">{@view~default.header.menuitem.admin.label@}</span></a>
```
Comment (rldhont) [inline lizmap/modules/lizmap/templates/user_menu.tpl]: Can you add phone-hidden class to the span text as connect text https://github.com/3liz/lizmap-web-client/pull/1180/files#diff-c6f3c6cf7ce683bf12c8bca40b3fab82R29
```
<span class="text hidden-phone">
```

Comment (nboisteault) [reply lizmap/modules/lizmap/templates/user_menu.tpl]: Same comment as @rldhont. The rest is ok for me.

Comment (laurentj) [reply lizmap/modules/lizmap/templates/user_menu.tpl]: added :-)

**Pull request merged**